snapcraft/internal/parser.py

```diff
@@ -51,9 +51,16 @@
 from collections import OrderedDict
 
 import snapcraft
-from snapcraft.internal import log, repo, sources
-from snapcraft.internal.errors import SnapcraftError, InvalidWikiEntryError
-from snapcraft.internal.project_loader import replace_attr
+from snapcraft.internal import log, project_loader, repo, sources
+from snapcraft.internal.errors import (
+    SnapcraftError,
+    InvalidWikiEntryError,
+    SnapcraftEnvironmentError,
+)
+from snapcraft.internal.project_loader import (
```

Comment:
why do you import project_loader above and then do this finer grained import instead of using `project_loader`?